Dockerfile

```diff
@@ -12,15 +12,14 @@ ADD setup.py /opt/atomicapp/
 ADD requirements.txt /opt/atomicapp/
 
 WORKDIR /opt/atomicapp
-
-RUN pip install .
+RUN python setup.py install
 
 WORKDIR /atomicapp
 VOLUME /atomicapp
-
-LABEL RUN docker run -it --rm --privileged --net=host -v `pwd`:/atomicapp -v /run:/run -v /:/host --name NAME -e NAME=NAME -e IMAGE=IMAGE IMAGE -v run /atomicapp
+ 
+LABEL RUN docker run -it --rm --privileged --net=host -v `pwd`:/atomicapp -v /run:/run -v /:/host --name NAME -e NAME=NAME -e IMAGE=IMAGE IMAGE -v RUNARGS run /atomicapp
```

Comment:
No doc yet, But we discussed this 
https://github.com/projectatomic/atomic/pull/65 Needs RUNARGS in the Dockerfile
i need it for putting -d option into install and run commands